src/test/java/seedu/guest/model/ModelManagerTest.java

```diff
@@ -36,15 +36,15 @@ public void setUserPrefs_nullUserPrefs_throwsNullPointerException() {
 
     @Test
     public void setUserPrefs_validUserPrefs_copiesUserPrefs() {
-        UserPrefs userPrefs = new UserPrefs();
-        userPrefs.setGuestBookFilePath(Paths.get("address/book/file/path"));
-        userPrefs.setGuiSettings(new GuiSettings(1, 2, 3, 4));
-        modelManager.setUserPrefs(userPrefs);
-        assertEquals(userPrefs, modelManager.getUserPrefs());
+        GuestPrefs guestPrefs = new GuestPrefs();
+        guestPrefs.setGuestBookFilePath(Paths.get("address/book/file/path"));
+        guestPrefs.setGuiSettings(new GuiSettings(1, 2, 3, 4));
+        modelManager.setUserPrefs(guestPrefs);
+        assertEquals(guestPrefs, modelManager.getUserPrefs());
 
         // Modifying userPrefs should not modify modelManager's userPrefs
-        UserPrefs oldUserPrefs = new UserPrefs(userPrefs);
-        userPrefs.setGuestBookFilePath(Paths.get("new/address/book/file/path"));
+        GuestPrefs oldUserPrefs = new GuestPrefs(guestPrefs);
+        guestPrefs.setGuestBookFilePath(Paths.get("new/address/book/file/path"));
```

Comment:
Should the file path be "new/guest/book/file/path" instead of "new/address/book/file/path"?